### ISSUE TITLE
Program 'flutter.bat' failed to run: The system cannot find the file specifiedAt line:1 char:1

### ISSUE THREAD

**Issue by aravindhnirmal** [flutter/flutter#130705]
<img width="732" alt="image" src="https://github.com/flutter/flutter/assets/97661995/d8c85fdb-5525-4578-a49b-f0bd5835eedf">
![image](https://github.com/flutter/flutter/assets/97661995/eb690fb7-a078-4adc-9502-de72b4067e87)
this is show when flutter doctor cmd gives

**Comment by dam-ease** [comment]
Hi @aravindhnirmal. Thanks for filing this issue. This doesn't seem to be an issue with flutter itself but can you provide more information on this:

- Does this occur when you newly installed flutter? If not, kindly provide more info on how you produced this?
- What version of flutter are you running this command on??

Thank you.

**Comment by aravindhnirmal** [comment]
yes in this first i download latest version of dart and next download the zip file of flutter latest version and next extract all and goto environmental variables(in win os 11) setting the path add new of path in it from the bin flutter and come to cmd type flutter doctor it trashed

**Comment by dam-ease** [comment]
Hi @aravindhnirmal. Thanks for your response. Please can you confirm you followed all the steps highlighted in the Getting Started Doc of Flutter https://docs.flutter.dev/get-started/install/windows??

**Comment by aravindhnirmal** [comment]
yes , same steps followed ,as same error show when i download the kotlin

**Comment by panulande** [comment]
I am also getting same issue, i've tried reinstalling and adding the Path variable but it keeps showing the same error.

**Comment by aravindhnirmal** [comment]
but i am tried in virtual box ubuntu it comes no error comes
